manifests/o/ojdkbuild/ojdkbuild/1.8.3321.9/ojdkbuild.ojdkbuild.installer.yaml

```
@@ -1,20 +1,20 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
-
-PackageIdentifier: ojdkbuild.ojdkbuild
-PackageVersion: 1.8.0.3321.9
-MinimumOSVersion: 10.0.0.0
-Installers:
-- Architecture: x64
-  InstallerType: wix
-  InstallerUrl: https://github.com/ojdkbuild/ojdkbuild/releases/download/java-1.8.0-openjdk-1.8.0.332-1.b09/java-1.8.0-openjdk-1.8.0.332-1.b09.ojdkbuild.windows.x86_64.msi
-  InstallerSha256: 84436C850A12F52C9658449D5BC16F1F6AD1C0AF820D510414A45B8D8AD64357
-  ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
-- Architecture: x86
-  InstallerType: wix
-  InstallerUrl: https://github.com/ojdkbuild/ojdkbuild/releases/download/java-1.8.0-openjdk-1.8.0.332-1.b09-x86/java-1.8.0-openjdk-1.8.0.332-1.b09.ojdkbuild.windows.x86.msi
-  InstallerSha256: 048b5de7c319a5989465b6d734f5f7a5a7a1eb0881f89e19473abe59fbd9c574
-  ProductCode: '{30DF20B4-B251-4DCF-AE84-59296ED8CB8D}'
-ManifestType: installer
-ManifestVersion: 1.2.0
-
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ojdkbuild.ojdkbuild
+PackageVersion: 1.8.3321.9
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://github.com/ojdkbuild/ojdkbuild/releases/download/java-1.8.0-openjdk-1.8.0.332-1.b09/java-1.8.0-openjdk-1.8.0.332-1.b09.ojdkbuild.windows.x86_64.msi
+  InstallerSha256: 84436C850A12F52C9658449D5BC16F1F6AD1C0AF820D510414A45B8D8AD64357
+  ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
```

```suggestion
  ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
  AppsAndFeaturesEntries:
  - DisplayName: OpenJDK 1.8.0_332-1-ojdkbuild
    ProductCode: '{F115813D-E05C-48A3-9F42-5C637056203B}'
```
For better upgrade matching